next-tavla/src/Admin/scenarios/BoardList/components/Row/index.tsx

```diff
@@ -47,6 +59,18 @@ function Row({ board }: { board: { id: string; settings?: TSettings } }) {
                     </IconButton>
                 </Tooltip>
             </Cell>
+            <Cell>
+                {board.settings?.meta?.dateModified &&
+                    toLocaleString(new Date(board.settings.meta.dateModified))}
+            </Cell>
+            <Cell>
+                {board.settings?.meta?.lastActive &&
+                    toLocaleString(new Date(board.settings.meta.lastActive))}
+            </Cell>
+            <Cell>
+                {board.settings?.meta?.created &&
+                    toLocaleString(new Date(board.settings.meta.created))}
+            </Cell>
```

Comment:
Tenker disse feltene kan droppes i "Mine tavler" visningen, de er primært for vårt eget bruk